framework/src/mfem/bcs/MFEMBoundaryCondition.C

```diff
@@ -14,7 +15,9 @@ MFEMBoundaryCondition::validParams()
   // Create user-facing 'boundary' input for restricting inheriting object to boundaries
   params.addParam<std::vector<BoundaryName>>(
       "boundary",
-      "The list of boundaries (ids or names) from the mesh where this boundary condition applies");
+      {"-1"},
```

Comment:
Does `-1` represent something to MFEM?

Reply:
Yes - it's the ID used to signify the boundary condition should be applied to every sideset on the mesh (although adding a comment that this is the default would be desirable here).

Reply:
Perfect, went ahead and added this